addon/options/options.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+</head>
+<body>
+	<form id="form">
+		<input type="text" name="separatorField" id="separatorField">
+		</textarea>
```

Comment:
I suppose the textarea isn't necessary

Reply:
Yeah, it's not supposed to be there. I had thought about working with a textarea, but decided on the input/form instead. I seem to have missed that closing textarea tag though.